test/test.ts

```diff
@@ -1436,6 +1436,22 @@ test('is.emptyStringOrWhitespace', t => {
 	});
 });
 
+test('is.nonEmptyStringAndNotWhitespace', t => {
+	t.false(is.nonEmptyStringAndNotWhitespace(' '));
+	t.false(is.nonEmptyStringAndNotWhitespace(String()));
+	t.true(is.nonEmptyStringAndNotWhitespace('🦄'));
```

Comment:
You should also test some non-string values

Reply:
Wrong assumption. All code has bugs, I actually expect it to be buggy, and someone can catch an edge case and fix it ;-)

Reply:
Added better test coverage